bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,138 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+## Supported Things
+
+| Thing Type ID | Devices |
+|------|---------------|
+|solarwatt:energymanager| EnergyManager itself
+|solarwatt:location| Location part of the EnergyManager 
+|solarwatt:pvplant| Power producing part of the EnergyManager
+|solarwatt:gridflow| Grid interaction part of the EnergyManager
+|solarwatt:inverter| inverter producing AC current; e.g. MyReserve, Fronius
+|solarwatt:batteryconverter| battery storage systems; e.g. MyReserve
+|solarwatt:powermeter| powermeters; e.g. S0BusCounter, MyReserve
+|solarwatt:evstation| electric-vehicle charging station; e.g. Keba Wallbox
+
+## Discovery
+
+You have to enter the hostname or ip-address of the energymanager itself.
+The attached devices and supported channels are discovered automatically.
+
+## Thing Configuration
+
+| Property | Default | Required | Description |
+|----------|---------|----------|-------------|
+| hostname | None | Yes | hostname or ip-address of the energy manager
+
+## Channels
+
```

Comment:
Can you add the channels in a table format? see other bindings for an example table layout and headings.